src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -45,7 +45,7 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
         }
 
         const paramsScript = this._getParamsScript(initializationData, cart);
-        const paypal = await this._paypalScriptLoader.loadPaypalCommerce(paramsScript);
+        const paypal = await this._paypalScriptLoader.loadPaypalCommerce(paramsScript, initializationData.isProgressiveOnboardingAvailable);
```

Comment:
minor, but for readability you could do something like
```js
const { useProgressiveOnboarding } = initializationData;
const paypal = await this._paypalScriptLoader.loadPaypalCommerce(paramsScript, useProgressiveOnboarding);
```

Reply:
@capsula4 I use `initializationData` for transmission  to `_getParamsScript` in 47 line too